### PR TITLE
Pools summary EP 

### PR DESCRIPTION
**Pull request by semyonoskin**

### CODE REVIEW

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/api/v1/endpoints/AmmStatsEndpoints.scala [hunk 1]
```diff
@@ -44,24 +44,32 @@ final class AmmStatsEndpoints(conf: RequestConfig) {
...
       .name("Pools statistic")
       .description("Get statistic about every known pool")
 
+  def getPoolsSummary: Endpoint[Unit, HttpError, List[PoolSummary], Any] =
+    baseEndpoint.get
+      .in(PathPrefix / "pools" / "summary")
+      .out(jsonBody[List[PoolSummary]])
+      .tag(Group)
+      .name("Pools summary")
+      .description("Get summary by every known pool")
```
Comment (GusevTimofey) [inline modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/api/v1/endpoints/AmmStatsEndpoints.scala:71]: write here about TVL sorting

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/db/sql/AnalyticsSql.scala [hunk 2]
```diff
@@ -25,7 +25,24 @@ final class AnalyticsSql(implicit lg: LogHandler) {
          |order by p.gindex desc limit 1;
          """.stripMargin.query[PoolSnapshot]
 
-  def getPoolSnapshots: Query0[PoolSnapshot] =
+  def getPoolSnapshots(hasTicker: Boolean): Query0[PoolSnapshot] = {
+
+    val assetJoin =
```
Comment (GusevTimofey) [inline modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/db/sql/AnalyticsSql.scala:30]: fmt

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/repositories/Pools.scala [hunk 3]
```diff
@@ -127,7 +127,7 @@ object Pools {
         _ <- trace"info(poolId=$poolId) -> ${r.size} info entities selected"
       } yield r
 
-    def snapshots: Mid[F, List[PoolSnapshot]] =
+    def snapshots(hasTicker: Boolean): Mid[F, List[PoolSnapshot]] =
```
Comment (GusevTimofey) [inline modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/repositories/Pools.scala:130]: add hasTicker into trace log

**Review by GusevTimofey — CHANGES_REQUESTED** [review]

File: modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/api/v1/services/AmmStats.scala [hunk 4]
```diff
@@ -109,8 +112,47 @@ object AmmStats {
...
+                PoolSummary(
+                  x.id,
+                  x.ticker.get,
+                  x.ticker.get,
+                  y.id,
+                  y.ticker.get,
+                  y.ticker.get,
+                  RealPrice
+                    .calculate(x.amount, x.decimals, y.amount, y.decimals)
+                    .setScale(6),
```
[the remark this answers is not in the document]
Comment (semyonoskin) [reply modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/api/v1/services/AmmStats.scala:140]: This is a method of RealPrice class : 

`@newtype case class RealPrice(value: BigDecimal) {
    def setScale(scale: Int): RealPrice = RealPrice(value.setScale(scale, RoundingMode.HALF_UP))
  }`

[the remark this answers is not in the document]
Comment (GusevTimofey) [reply modules/markets-api/src/main/scala/org/ergoplatform/dex/markets/api/v1/services/AmmStats.scala]: My bad

**Review by GusevTimofey — APPROVED** [review]

**Pull request merged**